compiler-core/src/javascript/expression.rs

```diff
@@ -247,6 +247,17 @@ impl<'module> Generator<'module> {
         result
     }
 
+    /// Wrap an expression in an immediately involked function expression if
+    /// required due to using early returns.
+    pub fn top_of_sequence_expression<'a>(&mut self, expression: &'a TypedExpr) -> Output<'a> {
+        match expression {
+            TypedExpr::Sequence { .. } | TypedExpr::Try { .. } => {
```

Comment:
Sequence here isn't strictly needed right? There are no other early return situations outside of `Try` right?

Reply:
I think you're right, yes. Thank you!